packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -739,6 +739,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                     logger.sendErrorEvent({ eventName: "SequenceNumberMismatch" }, error);
                 } else {
                     context.closeFn(error);
+                    context.disposeFn?.(error);
```

Comment:
Doubling up close+dispose here is essentially a back-compat measure for when close's semantics change, right? Might warrant a comment